airflow-core/src/airflow/api_fastapi/core_api/routes/public/extra_links.py

```diff
@@ -28,8 +28,8 @@
 from airflow.api_fastapi.core_api.datamodels.extra_links import ExtraLinkCollectionResponse
 from airflow.api_fastapi.core_api.openapi.exceptions import create_openapi_http_exception_doc
 from airflow.api_fastapi.core_api.security import DagAccessEntity, requires_access_dag
-from airflow.exceptions import TaskNotFound
 from airflow.models import DagRun
+from airflow.sdk.exceptions import TaskNotFound
```

Comment:
We shouldn't import exception from Task SDK in core and vice-versa

Reply:
Right. @uranusjr  is handling removing `models.DAG`  in https://github.com/apache/airflow/pull/54383